File: src/main/java/seremis/geninfusion/soul/gene/newAI/eatGrass/GeneAIEatGrass.java
package seremis.geninfusion.soul.gene.newAI.eatGrass;

import seremis.geninfusion.api.soul.IAllele;
import seremis.geninfusion.api.soul.lib.Genes;
import seremis.geninfusion.soul.MasterGene;
import seremis.geninfusion.soul.allele.AlleleBoolean;

public class GeneAIEatGrass extends MasterGene {

    public GeneAIEatGrass() {
        addControlledGene(Genes.GENE_AI_EAT_GRASS_INDEX);
    }

    @Override
    public Class<? extends IAllele> possibleAlleles() {
        return AlleleBoolean.class;
    }
}

File: src/main/scala/seremis/geninfusion/soul/gene/newAI/GeneAIEatGrass.scala
package seremis.geninfusion.soul.gene.newAI

import seremis.geninfusion.api.lib.Genes
import seremis.geninfusion.soul.MasterGene

class GeneAIEatGrass extends MasterGene(classOf[Boolean]) {
    addControlledGene(Genes.GeneAIEatGrassIndex)
}
